### PR TITLE
Fix errors encountered during setup process

### PR DESCRIPTION
**Pull request by colbyguan**
Hi there, I thought I'd drop a few fixes for issues I ran into while trying to set up the project:

This error appears on pip-installing the requirements:
```
Collecting forecastio (from -r requirements.txt (line 7))
  Could not find a version that satisfies the requirement forecastio (from -r requirements.txt (line 7)) (from versions: )
No matching distribution found for forecastio (from -r requirements.txt (line 7))
```
* I've updated the dependency to one what looks like a working, updated forecastio (now called [Dark Sky?](https://darksky.net/dev)) wrapper

Also got this error on app startup (OS X 10.13, Python 2.7.10):
```
  File "app.py", line 238
SyntaxError: Non-ASCII character '\xc2' in file app.py on line 238, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details
```
* Fixed by adding an encoding note at the top of `app.py`

I have also added more specific `virtualenv` instructions and Github's recommended `.gitignore` for python projects

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -83,7 +84,7 @@ def get_data_24h_in(byhour):
 
 app = dash.Dash()
 app.title = 'Weather || Astrool'
-image_filename = 'obj.png' # replace with your own image
+image_filename = 'logo.png'
```
Comment (colbyguan) [inline app.py:87]: This looks like the intended filename btw

File: README.rst [hunk 2]
```diff
@@ -14,7 +14,11 @@ Contains different interactive visualisations to understand better.
 
 How to Use:
 -----------
-Switch to a virtual environment! ``cd`` into project's directory.
+Switch to a virtual environment! ``cd`` into project's directory:
+```
+virtualenv --python=<path_to_python2> venv
```
Comment (colbyguan) [inline README.rst]: Wasn't sure if this project is intended to be Python 3 or 2, pls comment if change is needed

Comment (shreyasbapat) [reply README.rst]: Hi! Its a completely Python3 project!

File: app.py (hunk 1 above)
Comment (shreyasbapat) [reply app.py:87]: Oh thanks!

File: requirements.txt [hunk 3]
```diff
@@ -4,5 +4,4 @@ dash-html-components==0.11.0
 dash-core-components==0.24.1
 plotly
 geocoder
-forecastio
-...
+python-forecastio
```
Comment (shreyasbapat) [inline requirements.txt:7]: This was really important! Thanks a ton!

**Comment by shreyasbapat** [comment]
Thanks @colbyguan

**Pull request merged**